pipeline.yml

```diff
@@ -493,6 +495,11 @@ resources:
     secret_access_key: {{s3-bosh-releases-secret-access-key}}
     server_side_encryption: AES256
 
+- name: tests-timer
+  type: time
+  source:
+    interval: 30m
```

Comment:
I think we should probably run these more frequently? Every 10 minutes?

...and if we are running them at that rate, we should probably extend  https://github.com/18F/cg-pipeline-tasks/blob/master/bosh-errand.sh to support running errands with `--keep-alive` so we don't end up paying for 144 hours of VM time a day